src/common/pagination/Pagination.tsx

```diff
@@ -1,48 +1,69 @@
 import classNames from 'classnames';
 import React from 'react';
 import { useTranslation } from 'react-i18next';
-import ReactPaginate, { ReactPaginateProps } from 'react-paginate';
 import { IconArrowLeft, IconArrowRight } from 'hds-react';
 
 import Text from '../text/Text';
 import styles from './pagination.module.scss';
 
-export type PaginationProps = Pick<ReactPaginateProps, 'pageCount' | 'onPageChange' | 'initialPage' | 'forcePage'> & {
+export interface PaginationProps {
   className?: string;
-};
+  forcePage: number;
```

Comment:
What does `forcePage` mean? Would `initialPage` be a good term for this?

Reply:
It was a leftover from the `react-paginate` library, I renamed it to `pageIndex`